src/Utils/pfp-utils.js

```diff
@@ -0,0 +1,16 @@
+import rocket_red from "../images/rocket_red.svg";
+import rocket_may from "../images/rocket_may.svg";
+import rocket_serena from "../images/rocket_serena.svg";
+
+export const pfpPathToSvg = {
+  "rocket_red": rocket_red,
+  "rocket_may": rocket_may,
+  "rocket_serena": rocket_serena,
+};
+
+export const generatePfp = () => {
+  const pfpPool = ["rocket_may", "rocket_red", "rocket_serena"];
+  return pfpPool[Math.floor(Math.random() * pfpPool.length)];
```

Comment:
not fully sold on the random pfp... but ig it works with what little time we have